ansible/playbook.yaml

```
@@ -0,0 +1,68 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+
+  - name: trello_api_key
+    prompt: What is your Trello API Key? 
+
+  - name: trello_api_token
+    prompt: What is your Trello API Token?
+
+  tasks:
+  - name: Install the latest version of Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: latest
+
+  - name: Install the latest version of Python
+    become: yes
+    ansible.builtin.yum:
+      name: python311
+      state: latest
```

Not necessarily a suggestion, but worth noting that we can combine these:
```suggestion
  - name: Install the latest versions of Git & Python
    become: yes
    ansible.builtin.yum:
      name: 
        - git
        - python311
      state: latest
```
(Code not tested, could have typo)